app/controllers/tracks/concepts_controller.rb

```diff
@@ -9,7 +9,9 @@ def authenticated_index
     @concept_map_data = Track::DetermineConceptMapLayout.(@track)
 
     if current_user.joined_track?(@track)
-      @concept_map_data[:status] = UserTrack::GenerateConceptStatusMapping.(@user_track)
+      @concept_map_data = Track::DetermineConceptMapLayout.(@track).merge(
+        status: UserTrack::GenerateConceptStatusMapping.(@user_track)
+      )
```

Comment:
```suggestion
      @concept_map_data[:status] = UserTrack::GenerateConceptStatusMapping.(@user_track)
```
I think there was a merge conflict and I did something weird here, this is to fix.

Reply:
It should be this:
```suggestion
  @concept_map_data[:status] = UserTrack::GenerateConceptStatusMapping.(@user_track)
```

Reply:
(I've merged your suggestion before, but then it got overriden, so I'm leaving it so you can fix it 🙂)

Reply:
I feel like I've made this same change several times, I don't know where the commit keeps getting dropped :grinning: